projects/eval-ade7758/src/platform/platform.c

```diff
@@ -0,0 +1,113 @@
+/***************************************************************************//**
```

Comment:
This file seems to be specific to the Maxim platform. I'd say we need to have a `maxim` subdirectory.